api/system/v1alpha1/zone.proto

```diff
@@ -16,6 +16,8 @@ message Zone {
   option (kuma.mesh.resource).package = "system";
   option (kuma.mesh.resource).global = true;
   option (kuma.mesh.resource).ws.name = "zone";
+  // Zones are only managed globally
+  option (kuma.mesh.resource).kds = "model.ProvidedByGlobalFlag";
```

Comment:
Found it https://github.com/kumahq/kuma/pull/8584

Reply:
Ok so zone is synced from global to zone?

Reply:
what happens on standalone?

Reply:
It's created automatically on standalone with the name `default` https://github.com/kumahq/kuma/pull/8584/files#diff-55d71903df6fb064b2457cc866dcc4a4c379bb99b6ed37eee3475dc7925d74efR20

Reply:
fixing this then

Reply:
Got super confused for a long time here :)